src/components/Icons.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import CancelSvg from '../styles/CancelSvg';
+import InputIcon from '../styles/InputIcon';
+import SearchSvg from '../styles/SearchSvg';
+import { getClassName } from '../utils/helper';
+import Mic from './Mic';
+
+const CancelIcon = props => {
+  if (props.showClear) {
+    return props.clearIcon || <CancelSvg />;
+  }
+  return null;
+};
+
+const SearchIcon = props => {
+  if (props.showIcon) {
+    return props.icon || <SearchSvg />;
+  }
+  return null;
+};
+
+const Icons = props => {
+  const {
+    clearValue,
+    iconPosition,
+    showClear,
+    clearIcon,
+    theme,
+    currentValue,
+    handleSearchIconClick,
+    showIcon,
+    icon,
+    enableVoiceSearch,
+    innerClass,
+    renderMic,
+    getMicInstance,
+    onMicClick,
+    micStatus
+  } = props;
+
+  return (
+    <React.Fragment>
```

Comment:
Fragments were introduced in `v16`. So would this lib work with version below 16 like `15.6` or below ?

Reply:
In package.json it is `"react": "^15.0.0 || ^16.0.0"`, so
`yarn.lock` will be as shown below and so fragments are supported
![image](https://user-images.githubusercontent.com/10362927/65937353-44356a00-e43d-11e9-97be-5c0dbb07edf0.png)

Reply:
Didn't get you? 

Are you talking about `peerDependency`?

Reply:
In `devDependencies`, the version is >16: https://github.com/ShahAnuj2610/react-searchbox/blob/1d390fe6baa08f579551ffdd25b9fb922d488687/package.json#L63

Reply:
Just try the example with an older version of react if it works we are good to go